libs/apps/python.go

```diff
@@ -0,0 +1,127 @@
+package apps
+
+import (
+	"errors"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+const DEBUG_PORT = "5678"
+
+var defaultLibraries = []string{
+	"Flask==3.0.3",
+	"streamlit==1.38.0",
+	"gradio==4.44.0",
+	"uvicorn[standard]==0.30.6",
+	"databricks-sdk==0.33.0",
+	"databricks-sql-connector==3.7.3",
+	"debugpy",
+}
+
+type PythonApp struct {
+	config *Config
+	spec   *AppSpec
+	uvArgs []string
+}
+
+func NewPythonApp(config *Config, spec *AppSpec) *PythonApp {
+	return &PythonApp{config: config, spec: spec}
+}
+
+// PrepareEnvironment prepares the environment for running the app. It first checks if the app is in a virtual environment.
+// If not, it creates a virtual environment and installs the required libraries. It then installs the libraries from
+// requirements.txt if it exists.
+func (p *PythonApp) PrepareEnvironment() error {
+	// Create venv first
+	venvArgs := []string{"uv", "venv"}
+	if err := p.runCommand(venvArgs); err != nil {
+		return err
+	}
+
+	// Install default libraries
+	installArgs := append([]string{"uv", "pip", "install"}, defaultLibraries...)
+	if err := p.runCommand(installArgs); err != nil {
+		return err
+	}
+
+	// Install requirements if they exist
+	if _, err := os.Stat(filepath.Join(p.config.AppPath, "requirements.txt")); err == nil {
+		reqArgs := []string{"uv", "pip", "install", "-r", filepath.Join(p.config.AppPath, "requirements.txt")}
+		if err := p.runCommand(reqArgs); err != nil {
+			return err
+		}
+	}
+
+	// Set up run args
+	p.uvArgs = []string{"uv", "run"}
+	return nil
+}
+
+// GetCommand returns the command to run the app. If the spec has a command, it is returned.
+// If not, the function looks for a python file in the app directory and returns a command
+// to run that file. If the app is in a virtual environment, the command is modified to point
+// to the python binary in the virtual environment.
+func (p *PythonApp) GetCommand(debug bool) ([]string, error) {
+	spec := p.spec
+	// if no spec, find python file and use it to run app
+	if len(spec.Command) == 0 {
```

Comment:
The default behaviour should be app.py, not the first python file.
ref: https://docs.databricks.com/aws/en/dev-tools/databricks-apps/configuration#configuring-your-databricks-apps-with-the-appyaml-file